### PR TITLE
Lock sanity tests to use ubuntu 20.04 and not latest

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop / 8.0.x / 1.7.8.x
| Description?      | Please be specific when describing the PR. <br> Every detail helps: versions, browser/server configuration, specific module/theme, etc. Feel free to add more information below this table.
| Type?             | bug fix / improvement / new feature / refacto
| Category?         | FO / BO / CO / IN / WS / TE / LO / ME / PM
| BC breaks?        | yes / no
| Deprecations?     | yes / no
| Fixed ticket?     | Fixes #{issue number here}. If your PR fixes multiple issues, use Resolves #{issue number here}, Resolves #{another issue number here}.
| Related PRs       | If theme, autoupgrade or other module change is needed, provide a link to related PRs here.
| How to test?      | Please indicate how to best verify that this PR is correct.
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

### CODE REVIEW

File: .github/workflows/sanity.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     permissions:
       actions: write   #   to cancel/stop running workflows (styfle/cancel-workflow-action)
       contents: read   #   to fetch code (actions/checkout)
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (jolelievre) [inline .github/workflows/sanity.yml:11]: Same thing needs to be done for sanity-productV2.yml

Comment (jolelievre) [reply .github/workflows/sanity.yml:11]: Also, it's a temporary solution I guess? Do you know what is the root cause and what would be the ultimate fix?

**Review by jolelievre — CHANGES_REQUESTED** [review]

**Comment by matks** [comment]
A better fix was delivered on https://github.com/PrestaShop/PrestaShop/pull/30396

File: .github/workflows/sanity.yml (hunk 1 above)
Comment (matks) [reply .github/workflows/sanity.yml:11]: Ref https://github.com/PrestaShop/PrestaShop/pull/30396

**Pull request closed without merging**